src/inmanta/plugins.py

```diff
@@ -21,40 +21,35 @@
 import inspect
 import os
 import subprocess
+import typing
 import warnings
-from collections import abc
-from typing import TYPE_CHECKING, Any, Callable, Literal, Optional, Type, TypeVar
 
+import inmanta.ast as ast
 import inmanta.ast.type as inmanta_type
-from inmanta import const, protocol, util
-from inmanta.ast import LocatableString, Location, Namespace, Range, RuntimeException, TypeNotFoundException, WithComment
-from inmanta.ast.type import NamedType
-from inmanta.config import Config
-from inmanta.execute.proxy import DynamicProxy
-from inmanta.execute.runtime import QueueScheduler, Resolver, ResultVariable
-from inmanta.execute.util import NoneValue, Unknown
-from inmanta.stable_api import stable_api
-from inmanta.warnings import InmantaWarning
+import inmanta.warnings
+from inmanta import config, const, protocol, stable_api, util
+from inmanta.execute import proxy, runtime
+from inmanta.execute import util as execute_util
```

Comment:
this one shadows `inmanta.util` how to best handle this?

Reply:
I like it like this were we alias to `execute_util`.